eth/tracers/native/prestate.go

```diff
@@ -57,16 +57,14 @@ func newPrestateTracer(ctx *tracers.Context) tracers.Tracer {
 	return &prestateTracer{prestate: prestate{}}
 }
 
-func (t *prestateTracer) CapturePreEVM(env *vm.EVM, inputs map[string]interface{}) {
-	// TODO implement me? meowsbits/202203
-}
-
 // CaptureStart implements the EVMLogger interface to initialize the tracing operation.
 func (t *prestateTracer) CaptureStart(env *vm.EVM, from common.Address, to common.Address, create bool, input []byte, gas uint64, value *big.Int) {
 	t.env = env
 	t.create = create
 	t.to = to
 
+	t.lookupAccount(t.env.Context.Coinbase)
```

Comment:
What's this for?

Reply:
So as it includes the miner's prestate for that block. Miner prestate is useful for creating `stateDiff` tests. We can remove this (comment out) and use it only when needed by me.
When I will create this PR to upstream geth, I am going to question this line.